client/paths.go

```diff
@@ -41,7 +41,8 @@ const (
 	pathUserTeams                        = "/api/1/user/{userID}/teams"
 	pathUsers                            = "/api/1/users"
 	pathInvitation                       = "/api/1/invite/{inviteID}"
-	pathInvitations                      = "/api/1/team/{teamID}/invites"
+	pathInvitations                      = "/api/1/invites"
+	pathTeamInvitations                  = "/api/1/team/{teamID}/invites"
```

Comment:
We can probably remove this, unless we are still using this somewhere else in the provider

Reply:
nvm, i see we're still using it lol disregard!